bedrock/firefox/templates/firefox/enterprise/index.html

```diff
@@ -25,74 +23,242 @@
+            <h4 class="comparison-title-price">{{ _('Free') }}</h4>
+
+            <div class="comparison-cta cta-container mzp-c-button-download-container">
+              <a href="#download" class="mzp-c-button mzp-t-secondary mzp-t-product">{{ _('Download Firefox Browser') }}</a>
+              <a id="comparison-download-button" href="{{ url('privacy.notices.firefox') }}" class="mzp-c-button-download-privacy-link">{{ _('Firefox Privacy Notice') }}</a>
+            </div>
+
+            <ul class="comparison-list">
+              <li>{{ _('Public bug submission') }}</li>
+              <li>{{ _('Self-service knowledge base') }}</li>
+              <li>{{ _('Community support') }}</li>
+            </ul>
+          </section>
+
+          <section id="pager-page-premium" class="c-pager-page">
+            <h3 class="comparison-title-tier c-pager-hide-when-active">{{ _('Premium') }}</h3>
+            <h4 class="comparison-title-price">{{ _('Starting at $10 <span>per supported installation</span>') }}</h4>
+
+            <a id="comparison-sales-button" href="{{ url('firefox.enterprise.signup') }}" class="mzp-c-button mzp-t-product comparison-cta" data-cta-type="Enterprise-Sales" data-cta-position="Primary">{{ _('Contact Sales') }}</a>
+
+            <ul class="comparison-list">
+              <li>{{ _('Private bug submission') }}</li>
+              <li>{{ _('Critical security bug fixes with SLA') }}</li>
+              <li>{{ _('Concierge bug entry with guaranteed response time') }}</li>
+              <li>{{ _('Enterprise customer portal') }}</li>
+              <li>{{ _('Contribute to Firefox and our roadmap') }}</li>
+              <li>{{ _('Proactive notification on critical Firefox events') }}</li>
+              <li>{{ _('SLA management tool') }}</li>
+            </ul>
+          </section>
+        </div>
+
+      </div>
+    </section>
+  </div>
+
+  {% call call_out_compact(
+    title=_('Get info about Firefox premium support for your enterprise'),
+    desc=_('Contact our sales team for pricing and other details related to Firefox premium support.'),
+    class='mzp-t-product-firefox mzp-t-dark enterprise-section enterprise-callout'
+  ) %}
+    <a href="{{ url('firefox.enterprise.signup') }}" class="mzp-c-button mzp-t-product" data-cta-type="Enterprise-Sales" data-cta-position="Secondary">{{ _('Contact Sales') }}</a>
+  {% endcall %}
+
+  <div class="mzp-l-content">
+    <section id="download" class="enterprise-section enterprise-download">
+      <h2 class="enterprise-section-title">{{ _('Enterprise package downloads') }}</h2>
+
+      <ul class="mzp-l-card-third enterprise-download-lists">
+        {% call picto_card(
+          title=_('Windows 64-bit'),
+          class='enteprise-download-package win64',
+          heading_level=3,
+          custom_desc=True
+        ) %}
+          <ul class="mzp-u-list-styled">
+            <li>{{ _('The latest Firefox browser') }}</li>
+            <li>{{ _('The latest Firefox ESR') }}</li>
```

Comment:
Clay [had written this](https://docs.google.com/document/d/1nbSDIpMxmkLcy9Uts54IMB9LVZiTFGpwHRrC-_AFmD8/edit) as "Extended Support Release (ESR)" in the first column (64 bit) so we defined the abbreviation on the page the first time it appeared.

Reply:
@devinreams thank you for this. Please try and keep feedback consolidated in the gdoc, just so we can keep things centralised.